opm/autodiff/StandardWell_impl.hpp

```diff
@@ -621,6 +643,11 @@ namespace Opm
 
         // do the local inversion of D.
         invDuneD_[0][0].invert();
+
+        if ( param_.matrix_add_well_contributions_ )
```

Comment:
This is the part I referred to above; that could be removed and instead called from the BlackoilWellModel class.

Reply:
I disagree.